lib/f_service/result/base.rb

```diff
@@ -104,7 +117,13 @@ def on_success(*target_types)
       # @return [Success, Failure] the original Result object
       # @api public
       def on_failure(*target_types)
```

Comment:
I think the `unhandled` options should be a keyword argument, instead of a positional hash.
```suggestion
      def on_failure(*target_types, unhandled: true)
```

Reply:
Done: https://github.com/Fretadao/f_service/pull/30/commits/053339d645018720aba2692804f09ded79316c6d